fraud/requester.go

```diff
@@ -20,11 +21,12 @@ const (
 
 func (f *ProofService) requestProofs(
 	ctx context.Context,
+	id protocol.ID,
```

Comment:
One protocol ID for many proof types - we discussed in sync maybe having a default protocol ID (without a proof type) that is valid for all types

Reply:
`id` does not contain proofType. So, we will request all available fraud proofs.